tsl/test/sql/compression_ddl.sql

```diff
@@ -465,5 +465,20 @@ ALTER TABLE i2844 SET (timescaledb.compress = FALSE);
 -- compression alter schema tests
 \ir include/compression_alter.sql
 
-
+-- test compression default handling
```

Comment:
The alter tests are in include/compression_alter.sql. Could you please move this test to that file?

Reply:
changed